subworkflows/local/prepare_genome.nf

```diff
@@ -102,6 +105,17 @@ workflow PREPARE_GENOME {
             ch_bwa_index = BWAMEM2_INDEX ( ch_fasta ).index
             ch_versions  = ch_versions.mix(BWAMEM2_INDEX.out.versions)
         }
+    } else if ('dragmap' in prepare_tool_indices) {
+        if (params.dragmap) {
+            if (params.dragmap.endsWith('.tar.gz')) {
+                ch_bwa_index = UNTAR_DRAGMAP_INDEX ( params.dragmap ).untar
```

Comment:
😱 I don't think we are untarring our index. Did you update it in igenomes

Reply:
This is just in case a user wants to feed in a tarred up index. I want to refactor the whole prepare genome to be similar to sarek because it's easier to follow, but I'm resisting the urge in order to release v2.

Reply:
https://github.com/nf-core/nascent/issues/60#issue-1370131985